validator/rpc/wallet.go

```diff
@@ -304,6 +305,11 @@ func (s *Server) ImportAccounts(
 	if err != nil {
 		return nil, err
 	}
+	for _, stat := range statuses {
+		if stat.Status == ethpbservice.ImportedKeystoreStatus_ERROR {
+			return nil, status.Error(codes.FailedPrecondition, stat.Message)
```

Comment:
Once again, this will return a 412. Is this safe to do according to spec?

Reply:
see previous message